server/model/events/events.go

```diff
@@ -158,51 +158,6 @@ func TriggergRPCUnreachableHostError(testID id.ID, runID int, err error) model.T
 	}
 }
 
-func TraceFetchingStart(testID id.ID, runID int) model.TestRunEvent {
-	return model.TestRunEvent{
-		TestID:              testID,
-		RunID:               runID,
-		Stage:               model.StageTrace,
-		Type:                "FETCHING_START",
-		Title:               "Starting the trace fetching process",
-		Description:         "Starting the trace fetching process",
-		CreatedAt:           time.Now(),
-		DataStoreConnection: model.ConnectionResult{},
-		Polling:             model.PollingInfo{},
-		Outputs:             []model.OutputInfo{},
-	}
-}
-
-func TraceQueuedInfo(testID id.ID, runID int) model.TestRunEvent {
-	return model.TestRunEvent{
-		TestID:              testID,
-		RunID:               runID,
-		Stage:               model.StageTrace,
-		Type:                "QUEUED_INFO",
-		Title:               "Trace Run has been queued to start the fetching process",
-		Description:         "Trace Run has been queued to start the fetching process",
-		CreatedAt:           time.Now(),
-		DataStoreConnection: model.ConnectionResult{},
-		Polling:             model.PollingInfo{},
-		Outputs:             []model.OutputInfo{},
-	}
-}
-
-func TraceDataStoreConnectionInfo(testID id.ID, runID int) model.TestRunEvent {
-	return model.TestRunEvent{
-		TestID:              testID,
-		RunID:               runID,
-		Stage:               model.StageTrace,
-		Type:                "DATA_STORE_CONNECTION_INFO",
-		Title:               "A Data store connection request has been executed,test connection result information",
-		Description:         "A Data store connection request has been executed,test connection result information",
-		CreatedAt:           time.Now(),
-		DataStoreConnection: model.ConnectionResult{},
-		Polling:             model.PollingInfo{},
-		Outputs:             []model.OutputInfo{},
-	}
-}
```

Comment:
This event is done at the same time we are fetching a trace from the DataStore. Thinking about that, I've focused on emitting the `TraceFetchingStart` event.